sys/src/9/amd64/devacpi.c

```diff
@@ -1915,6 +1915,7 @@ static void acpiinitonce(void)
 	 * We are not ready in this kernel for that.
 	 */
 	assert(fadt != nil);
+#if 0	/* These iomaps intefere with acpica library */
```

Comment:
Normally I don't like leaving #if 0 in code but this bit of acpi is still not well worked out. So it's fine.